cypress/integration/plugins/security-dashboards-plugin/change_tenant_successfully.js

```diff
@@ -0,0 +1,65 @@
+/*
+
+* Copyright OpenSearch Contributors
+
+* SPDX-License-Identifier: Apache-2.0
+
+*/
+
+import { CURRENT_TENANT } from '../../../utils/commands';
+import tenantDescription from '../../../fixtures/plugins/security-dashboards-plugin/tenants/testTenant.json';
+
+const tenantName = 'test';
+
+if (Cypress.env('SECURITY_ENABLED')) {
+  describe('Switch tenants when visiting copyed links: ', () => {
+    before(() => {
+      cy.server();
+
+      cy.createTenant(tenantName, tenantDescription);
+    });
+    if (Cypress.browser.name === 'chrome') {
+      it('Checks that the tenant switcher can switch tenants despite a different tenant being present in the tenant query parameter.', function () {
+        CURRENT_TENANT.newTenant = 'private';
+        function switchTenantTo(newTenant) {
```

Comment:
Would this function be useful in other tests or is this specific to this test?

Reply:
It has now been extracted for use in other cypress tests.